graphql_compiler/compiler/ir_lowering_sql/__init__.py

```diff
@@ -223,3 +231,20 @@ def _map_block_index_to_location(ir_blocks):
                 block_index_to_location[ix] = ir_block.location
             current_block_ixs = []
     return block_index_to_location
+
+
+def lower_unary_transformations(ir_blocks):
+    """Raise exception if any unary transformation block encountered."""
+    def visitor_fn(expression):
+        """Raise error if current expression is a UnaryTransformation."""
+        if not isinstance(expression, expressions.UnaryTransformation):
+            return expression
+        raise NotImplementedError(
+            u'"has_edge_degree" filter operation is unsupported by SQL backend.'
```

Comment:
I find this a bit misleading. When this error gets hit, we've found a `UnaryTransformation` expression. While it might currently be the case that `has_edge_degree` is the only reason that such an expression might be generated at the moment, I don't think we should assume this will continue to always be the case. I'd make the error message complain about the specific expression that it encountered, and include it in the message.

Reply:
Fixed, the relevant expression is now output in the error